packages/core/src/utils/CollectLabels.ts

```diff
@@ -257,9 +262,13 @@ const measureTextContext = measureTextElement.getContext("2d")!;
  * @returns `TextMeasurement` object and includes data such as `width` and `height` of the text.
  */
 export function measureText(text: string, font: string): TextMeasurement {
+  const measureTextElement = document.createElement("canvas");
+  const measureTextContext = measureTextElement.getContext("2d")!;
   measureTextContext.textBaseline = "alphabetic";
   measureTextContext.font = font;
   const measurements = measureTextContext.measureText(text);
+  // TODO: removing the element causes issues of "NotFoundError: The node to be removed is not a child of this node."
+  // document.removeChild(measureTextElement);
```

Comment:
This makes sense, right? Calling [`document.createElement`](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement) just creates a DOM node, it doesn't actually attach it as a child of `document` as far as I know.

Reply:
Right. I meant to call `remove()` instead. Seems to work just fine.